src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -1086,27 +980,148 @@ class PyTiledGumpPic(PyBaseControl):
-class PyTTFTextInputField(PyBaseControl):
+class PySoundEntry:
     ""
-    Text: str = None
-    CaretIndex: int = None
-    NumbersOnly: bool = None
-    AcceptKeyboardInput: bool = None
-    ConvertHtmlColors: bool = None
+    ID: int = None
+    X: int = None
+    Y: int = None
+    Time: datetime = None
```

Ruff (0.15.1)

[error] Undefined name `datetime`

(F821)